SECURITY.md

```diff
@@ -1,13 +1,5 @@
 # Security Policy
 
-## Supported Versions
```

Comment:
I: We will be discussing our formal security policy next week and will codify the outcome of that conversation here.